samples/bmi/pom.xml

```diff
@@ -69,6 +69,7 @@
       <plugin>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
+        <version>0.13</version>
```

Comment:
It's not a good practice to specify the plugin version here.

Reply:
Why? Any suggestions?

Reply:
you can use the property variable to manage the version in the single place, just like put the <version>${rat.version}</version> here.